### PR TITLE
Added support for Tuya Zigbee ultrasonic water meter.

### PR DESCRIPTION
**Pull request by keskival**
Added support for Tuya Zigbee ultrasonic water meter.

Screenshot from 2024-12-26 13-12-53

### CODE REVIEW

File: custom_components/tuya_local/devices/tuya_zigbee_ultrasonic_watermeter.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: Water meter
+products:
+  - id: zlwr0raf
+    name: Smart Water Meter Zigbee
```
Comment (make-all) [inline custom_components/tuya_local/devices/tuya_zigbee_ultrasonic_watermeter.yaml:4]: This should be linked to a specific product, not a generic description.

Comment (keskival) [reply custom_components/tuya_local/devices/tuya_zigbee_ultrasonic_watermeter.yaml]: Added "Tuya". This product is known by many names, none of them more specific than this. I think it has been translated from Chinese by several different companies.

**Comment by keskival** [comment]
https://www.tuyaexpo.com/mobile/product/898043

File: custom_components/tuya_local/devices/tuya_zigbee_ultrasonic_watermeter.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: Water meter
 products:
   - id: zlwr0raf
-    name: Tuya Smart Water Meter Zigbee
+    name: Tuya 214C Valve Control Ultrasonic Water Meter Zigbee
```
Comment (make-all) [inline custom_components/tuya_local/devices/tuya_zigbee_ultrasonic_watermeter.yaml:4]: According to the link, the manufacturer is ZPMeter, not Tuya.

**Pull request merged**